src/codegen/llvm/codegen_llvm_visitor.cpp

```diff
@@ -65,6 +65,50 @@ static bool can_vectorize(const ast::CodegenForStatement& statement, symtab::Sym
     return unsupported.empty() && supported.size() <= 1;
 }
 
+#if LLVM_VERSION_MAJOR >= 13
+void CodegenLLVMVisitor::add_vectorizable_functions_from_vec_lib(llvm::TargetLibraryInfoImpl& tli,
+                                                                 llvm::Triple& triple) {
+    // Since LLVM does not support SLEEF as a vector library yet, process it separately.
+    if (vector_library == "SLEEF") {
+        // Populate function definitions for AArch64 architecture.
+#define FIXED(w) llvm::ElementCount::getFixed(w)
+#define DISPATCH(func, vec_func, width) {func, vec_func, width},
+        const llvm::VecDesc functions[] = {
+            DISPATCH("llvm.exp.f32", "_ZGVnN4v_expf", FIXED(4))
+            DISPATCH("llvm.exp.f64", "_ZGVnN2v_exp", FIXED(2))
+            DISPATCH("llvm.pow.f32", "_ZGVnN4vv_powf", FIXED(4))
+            DISPATCH("llvm.pow.f64", "_ZGVnN2vv_pow", FIXED(2))
+        };
```

Comment:
Just as a future reference: `Vector Function Name Mangling` conventions are documented [here](https://sourceware.org/glibc/wiki/libmvec?action=AttachFile&do=view&target=VectorABI.txt) and see https://github.com/shibatch/sleef/issues/13.